Provide support for scraping metrics for Azure ExpressRoute circuits (#1254)
tomkerkhove/promitor: docs/configuration/v2.x/metrics/express-route-circuits.md

---
layout: default

> title: Azure Express Route Circuits Declaration

Reviewer: ```suggestion
title: Azure Express Route Circuit Declaration
```

---

> ## Azure Express Route Circuits

Reviewer: ```suggestion
## Azure Express Route Circuit
```

> ![Availability Badge](https://img.shields.io/badge/Available%20Starting-v1.0-green.svg)![Resource Discovery Support Badge](https://img.shields.io/badge/Support%20for%20Resource%20Discovery-Yes-green.svg)

Reviewer: This will only be part of v2.0
```suggestion
![Availability Badge](https://img.shields.io/badge/Available%20Starting-v2.0-green.svg)![Resource Discovery Support Badge](https://img.shields.io/badge/Support%20for%20Resource%20Discovery-Yes-green.svg)
```

> You can declare to scrape an Azure Express Route Circuits (without Peerings) via the `ExpressRouteCircuits` resource

Reviewer: We typically use name for one instance
```suggestion
You can declare to scrape an Azure Express Route Circuit (without Peerings) via the `ExpressRouteCircuit` resource
```

type.

The following fields need to be provided:

> - `expressRouteCircuitsName` - The name of the express route circuits

Reviewer: I'm new to ExpressRoute but is this the name of the express route or expressroute circuit?

Reviewer: ```suggestion
- `expressRouteCircuitName` - The name of the Azure Express Route circuit
```

Reviewer: Actually since it's the `ExpressRouteCircuit` scraper I'm even wondering if we can't call it `CircuitName`?

Author: I missed that. It's the name of the ExpressRoute Circuit

Author: Regarding the name change I'm pretty unemotional. The full name might be a bit clearer
